python/docs/source/index.md

```diff
@@ -3,94 +3,119 @@ hide-toc: true
 html_theme.sidebar_secondary.remove: true
 title: Kaskada Timestreams
 ---
+# Introduction
 
 <div class="px-4 py-5 my-5 text-center">
     <img class="d-block mx-auto mb-4" src="_static/kaskada.svg" alt="" width="auto">
-    <h1 class="display-5 fw-bold">Event-processing for AI applications.</h1>
+    <h1 class="display-5 fw-bold">Real-Time AI without the fuss.</h1>
     <div class="col-lg-7 mx-auto">
-      <p class="lead mb-4">Next-generation, real-time and historic event processing.
+      <p class="lead mb-4">Kaskada is a simple streaming engine that connects AI models to real-time & historical data.
       </p>
     </div>
 </div>
 
+(what-it-provides)=
+## Kaskada completes the Real-Time AI stack, providing...
+
 ```{gallery-grid}
 :grid-columns: 1 2 2 3
 
-- header: "{fas}`timeline;pst-color-primary` Real-time processing for all"
-  content: "Quickly process events so you can respond in real-time."
-  link: ".#stream"
-- header: "{fab}`python;pst-color-primary` Python-native"
-  content: "Use Python so you can load data, process it, and train and serve models from one place."
-  link: ".#python"
-- header: "{fas}`gauge-high;pst-color-primary` Get started immediately"
-  content: "No infrastructure to provision let's you jump right in."
-  link: ".#get-started"
-
-- header: "{fas}`fast-forward;pst-color-primary` Real-time, Batch and Streaming"
-  content: "Execute large-historic queries or materialize in real-time. Or both."
-  link: ".#real-time-and-historic"
-- header: "{fas}`rocket;pst-color-primary` Local, Remote and Distributed"
-  content: "Develop and test locally. Deploy to Docker, K8s or a service for production."
-  link: ".#local-and-distributed"
-- header: "{fas}`backward;pst-color-primary` Time-travel"
-  content: "Generate training examples from the past to predict the future."
-  link: ".#time-travel"
+- header: "{fas}`timeline;pst-color-primary` Stream Processing"
```

Comment:
(1): I experimented with some variations along these lines, but I feel like "features" is too data-science-specific, and "prompts" or "contexts" also feel like jargon.  What do you think of "Real-time Aggregation"? It's a more general term than the other three, and also communicates that we're doing more than just event-level mapping.

(2): Agreed

(3): My concern is that we haven't found a good catchphrase for this. "Time-travel" is all over the place and doesn't really mean what we mean. "Historic computation" is better, but still seems sort of vague. What I like about "historic replay" is that it's describing the process and I feel like it does a better job suggesting the potential applications.

I asked ChatGPT what it thought the two terms meant:

> "Historic computation" and "historic replay" seem to be related terms, but they likely have distinct meanings based on their usage. Here's how they might differ:
> 
> 1. **Historic Computation**:
>    - This term could refer to the process of performing computations or calculations using methods and techniques that were prevalent in the past. It might involve using historical computing devices, algorithms, and tools that were available before the modern digital era.
>    - Historic computation could also encompass the study of how computations were done in the past, including the evolution of mathematical and computational techniques.
> 
> 2. **Historic Replay**:
>    - This term could refer to the act of replaying or recreating historical events, scenarios, or processes using computational models, simulations, or data analysis techniques.
>    - Historic replay might involve using available historical data to simulate and visualize past events, allowing researchers to gain insights into the dynamics and outcomes of those events.
>    - In certain contexts, historic replay could also refer to the process of reviewing and analyzing historical records, logs, or data to reconstruct past sequences of events, often in fields like computer science, where system behavior needs to be understood retrospectively.
> 
> In summary, while both terms involve the concept of history and computation, "historic computation" seems more focused on the computation methods of the past, while "historic replay" appears to center on recreating or analyzing historical events using computational techniques. The exact meanings would depend on the specific context in which these terms are used.